values-changes.yaml

```
@@ -378,3 +378,5 @@ changes:
   - version: 44
     additions:
       - cluster.defaultStorageClass: ''
+    deletions:
```

I think we would need a new version here
```suggestion
  - version: 45
    deletions:
```